src/util-spm.c

```diff
@@ -403,6 +400,9 @@ static uint8_t *BoyerMooreNocaseWrapper(uint8_t *text, uint8_t *in_needle, int t
 }
 
 #ifdef ENABLE_SEARCH_STATS
+/* Number of times to repeat the search (for stats) */
```

Comment:
is this related to the lua work?

Reply:
No;  There were warnings issued while rebuilding the Lua changes so I included that commit with the Lua changes.

Reply:
I've submitted the fixup changes in another PR and will drop them.